language/compiler/bytecode_source_map/src/disassembler.rs

```diff
@@ -132,11 +224,219 @@ impl<Location: Clone + Eq + Default> Disassembler<Location> {
             .zip(kinds.iter())
             .map(|((name, _), kind)| format!("{}: {:#?}", name.as_str(), kind))
             .collect();
-        if ty_params.is_empty() {
-            "".to_string()
-        } else {
-            format!("<{}>", ty_params.join(", "))
+        Self::enclose_ty_params(ty_params)
+    }
+
+    fn disassemble_bytecode(
+        &self,
+        function_definition_index: FunctionDefinitionIndex,
+    ) -> Result<String> {
+        if !self.options.print_code {
+            return Ok("".to_string());
         }
+
+        let function_def = self.get_function_def(function_definition_index)?;
+        let locals_sigs = self
+            .source_mapper
+            .bytecode
+            .locals_signature_at(function_def.code.locals);
```

Comment:
I wonder if we should also print out the locals. Personally it's a bit weird to see names of the locals appear in instructions without a list of them at the beginning. Thoughts?

Reply:
I think printing out the locals is a good idea.